image-build-push.sh

```diff
@@ -0,0 +1,36 @@
+# Terminal output colors 
+GREEN='\033[0;32m'
+ENDCOLOR='\033[0m' # No Color
+RED='\033[0;31m'
+
+# Environment Variables 
+PREFER_PODMAN="${PREFER_PODMAN:-0}"
+PUSH="${PUSH:-1}"
+TAG="${TAG:-test}"
+REGISTRY_ORG="${REGISTRY_ORG:-jezhu}"
```

Comment:
```suggestion
REGISTRY_ORG="${REGISTRY_ORG:-openshift-observability-ui}"
```

Reply:
Because this file will be used for development purposes I'll change this to `<your-repository>` to clarify that it should be pushed to the developer's repository during development.

Reply:
I'll make a note in the `README.md` about this.